sycl/doc/extensions/experimental/sycl_ext_oneapi_enqueue_functions.asciidoc

```diff
@@ -709,6 +709,32 @@ optimize such partial barriers.
 _{endnote}_]
 |====
 
+==== Command Graph
+
+The functions in this section are only available if the
+link:./sycl_ext_oneapi_graph.asciidoc[
+  sycl_ext_oneapi_graph] extension is supported.
+
+|====
+a|
+[frame=all,grid=none]
+!====
+a!
+[source,c++]
+----
+namespace sycl::ext::oneapi::experimental {
+
+template <typename CommandGroupFunc>
+void submit(command_graph<graph_state::modifiable> g, CommandGroupFunc&& cgf);
+
+}
+----
+!====
+_Effects_: Submits a command-group function object (as defined by the SYCL
+specification) to the `command_graph` for deferred execution. Subsequent calls
+will create a linear chain of nodes with implicit edges.
```

Comment:
I'm not sure about these linear semantics, if a buffer accessor is used in the command-group that could create edges to nodes to other predecessor.

"subsequent" also doesn't take into account interleaving other calls to the graph APIs which add nodes in between two calls to this API.

We're using `depends_on_all_leaves` to implement this, I think we could use those semantics in this definition.